vue-i18n does not work with Nuxt 3.7.2 without transpiling

### Reporting a bug?

Integrating vue-i18n into a Nuxt 3.7 application according to the instructions results in the following error when building and running the application:

`[nuxt] [request error] [unhandled] [500] Cannot find module '/home/projects/github-hnz7et-dysose/.output/server/chunks/dist/vue-i18n.runtime.mjs' imported from /home/projects/github-hnz7et-dysose/.output/server/chunks/app/server.mjs`

It works fine in dev.

### Expected behavior

No error.

### Reproduction

https://stackblitz.com/edit/github-hnz7et-dysose?file=nuxt.config.ts

### System Info

```shell
System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 16.20.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.4.2 - /usr/local/bin/npm
    pnpm: 8.6.10 - /usr/local/bin/pnpm
  npmPackages:
    @intlify/unplugin-vue-i18n: ^1.2.0 => 1.2.0 
    vue-i18n: ^9.4.0 => 9.4.0 
```

### Screenshot

_No response_

### Additional context

It might be just a documentation error according Daniel Roe from Nuxt: https://github.com/nuxt/nuxt/issues/23180#issuecomment-1717393059

This workaround seems to work without issues.
```diff
  // https://nuxt.com/docs/api/configuration/nuxt-config
  import VueI18nPlugin from '@intlify/unplugin-vue-i18n';
  
  export default defineNuxtConfig({
    devtools: { enabled: true },
+   build: {
+     transpile: [/vue-i18n/]
+   },
    vite: {
      plugins: [
        VueI18nPlugin.vite({
          /* options */
  
        }),
      ],
    },
  });
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't [already an issue](https://github.com/intlify/vue-i18n-next/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussions](https://github.com/intlify/vue-i18n-next/discussions)

--- a/docs/guide/integrations/nuxt3.md
+++ b/docs/guide/integrations/nuxt3.md
@@ -271,6 +271,9 @@ import VueI18nVitePlugin from '@intlify/unplugin-vue-i18n/vite' // [!code ++]
 
 // https://nuxt.com/docs/api/configuration/nuxt-config
 export default defineNuxtConfig({
+  build: { // [!code ++]
+    transpile: ['vue-i18n'] // [!code ++]
+  }, // [!code ++]
   vite: { // [!code ++]
     plugins: [ // [!code ++]
       VueI18nVitePlugin({ // [!code ++]

